_posts/developers/pod-server/2019-01-02-00_apache-config.md

```diff
@@ -75,9 +79,7 @@ ProxyPassReverse / https://localhost:8443/
 </VirtualHost>
 ```
 
-* edit default-ssl.conf<br />
-`$ nano /etc/apache2/sites-available/default-ssl.conf`<br />
-add to config in the appropriate section as follows<br />
+* edit the appropriate sections of `/etc/apache2/sites-available/default-ssl.conf` as follows:
```

Comment:
Should we mention that the example config uses `example.org` and as such, people should substitute their server?